node/src/node/isometrics.cpp

```diff
@@ -0,0 +1,528 @@
+#include <tuple>
+#include <cassert>
+
+#include "isometrics.hpp"
+
+#include <boost/range/adaptors.hpp>
+#include <boost/range/algorithm.hpp>
+#include <boost/range/numeric.hpp>
+
+// #define ISOMETRICS_DEBUG
+#undef ISOMETRICS_DEBUG
+
+#ifdef ISOMETRICS_DEBUG
+#define dbg(msg) std::cerr << msg << '\n'
+#define DBG_DUMP_UUIDS(os, logo, container) aux::dump_uuids(os, logo, container)
+#else
+#define dbg(msg)
+#define DBG_DUMP_UUIDS(os, logo, container)
+#endif
+
+namespace cocaine {
+namespace detail {
+namespace service {
+namespace node {
+
+namespace ph = std::placeholders;
+
+namespace conf {
+    constexpr auto PURGATORY_QUEUE_BOUND = 8 * 1024;
+    constexpr auto SHOW_ERRORS_LIMIT = 3;
+
+    // <name, is metric monotonically increasing per application (aggregate) level>
+    const std::vector<std::pair<std::string, bool>> counter_metrics_names =
+    {
+        // yet abstract cpu load measurement
+        { "cpu", false},
```

Comment:
Use enums instead of bool. It improves readability. Thus, there will be no need for separate comment.